mcs/class/System/System.Net.NetworkInformation/UnixIPGlobalProperties.cs

```diff
@@ -280,6 +280,31 @@ void GetRows (string file, List<string []> list)
 			}
 		}
 
+		TcpState UnixTcpStateToTcpState (int unixState)
+		{
+			//The values of these states in Linux are listed here:
+			//https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/net/tcp_states.h?id=HEAD
+			TcpState [] states = {
```

Comment:
Would be better to have just have a switch statement, this way it has to allocate an array